libdebug/debugger/internal_debugger.py

```diff
@@ -189,14 +197,14 @@ def clear(self: InternalDebugger) -> None:
 
     def start_up(self: InternalDebugger) -> None:
         """Starts up the context."""
-
         # The context is linked to itself
         link_to_internal_debugger(self, self)
 
         self.start_processing_thread()
         with extend_internal_debugger(self):
             self.debugging_interface = provide_debugging_interface()
-            self.memory = MemoryView(self._peek_memory, self._poke_memory)
+            self._fast_memory = DirectMemoryView(self._fast_read_memory, self._fast_write_memory)
```

Comment:
do they really need to be linked to the internal_debugger? Do they use it (despite the two methods passed ar arguments)?